include/seqan3/range/view/translation.hpp

```diff
@@ -33,256 +33,567 @@
- * |---------------------|---------------------------------------|-----------------------------------------------------------|
- * | range               | seqan3::input_range_concept           | seqan3::view_concept + all range concepts met by `irng_t` |
- * | `range_reference_t` | seqan3::nucleotide_concept            |                                                           |
+ * |                     | `urng_t` (underlying range type)                                   | `rrng_t` (range return type)                                |
+ * |---------------------|--------------------------------------------------------------------|-------------------------------------------------------------|
+ * | range               | seqan3::sized_range_concept && seqan3::random_access_range_concept | seqan3::view_concept && seqan3::random_access_range_concept |
+ * | `range_reference_t` | seqan3::nucleotide_concept                                         | seqan3::aa27                                                |          |
  *
- * * `irng_t` is the type of the range modified by this view (input).
+ * * `urng_t` is the type of the range modified by this view (input).
  * * `rrng_type` is the type of the range returned by this view.
  * * for more details, see \ref view.
  *
- * \par Example
+ * ### Example
  *
  * Operating on a range of seqan3::dna5:
  * ```cpp
  * dna5_vector vec{"ACGTACGTACGTA"_dna5};
  *
- * // single frame translation
- * auto v1 = vec | view::translate_single;                     // == [T,Y,V,R]
+ * // First forward frame
+ * auto v1 = vec | view::translate_single(translation_frames::FIRST_FRAME_FORWARD);                                  // == [T,Y,V,R]
+ *
+ * // First reverse frame
+ * auto v2 = vec | view::translate_single(translation_frames::FIRST_FRAME_REVERSE);                                  // == [Y,V,R,T]
+ *
+ * // Second forward frame
+ * auto v3 = vec | view::translate_single(translation_frames::SECOND_FRAME_FORWARD);                                 // == [R,T,Y,V]
+ *
+ * // Second reverse frame
+ * auto v4 = vec | view::translate_single(translation_frames::SECOND_FRAME_REVERSE);                                 // == [T,Y,V,R]
+ *
+ * // Third forward frame
+ * auto v5 = vec | view::translate_single(translation_frames::THIRD_FRAME_FORWARD);                                  // == [V,R,T]
+ *
+ * // Third reverse frame
+ * auto v6 = vec | view::translate_single(translation_frames::THIRD_FRAME_REVERSE);                                  // == [R,T,Y]
  *
  * // function syntax
- * auto v2 = view::translate_single(vec);                      // == [T,Y,V,R]
+ * auto v7 = view::translate_single(vec, translation_frames::FIRST_FRAME_FORWARD);                                   // == [T,Y,V,R]
  *
  * // combinability
- * auto v3 = vec | view::complement | view::translate_single;  // == [C,M,H,A]
+ * auto v8 = vec | view::complement | view::translate_single(translation_frames::FIRST_FRAME_REVERSE);               // == [M,H,A,C]
```

Comment:
and because of our new functor generation template, you should also be able to
```cpp
auto v8 = vec | view::complement | view::translate_single(translation_frames::FIRST_FRAME_REVERSE); 
auto v3 = vec | view::complement | view::translate_single();  // default frame
auto v3 = vec | view::complement | view::translate_single;  // also works for default frame

Reply:
also please double-check the enum labels in the documentation here! Best copy'n'paste from you tests in the end.